appservice/src/createAppService/SiteRuntimeStep.ts

```diff
@@ -43,6 +43,9 @@ export class SiteRuntimeStep extends AzureWizardPromptStep<IAppServiceWizardCont
                     data: rt
                 };
             });
+
+            // filters out Node 4.x and 6.x as they are EOL
+            runtimeItems = this.filterRuntimes(runtimeItems, /node\|(4|6)\./);
```

Comment:
The `filterRuntimes` function seems unnecessary. It can already be a one-liner:
```
runtimeItems = runtimeItems.filter(qp => !/node\|(4|6)\./i.test(qp.data.name));
```

(Remember I turned off typedef for arrow-parameter. Looks much cleaner without the typedef IMO)